lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -56,7 +56,7 @@
                     );
                     this.discussionView = new TeamDiscussionView({
                         el: this.$('.discussion-module'),
-                        readOnly: this.$('.discussion-module').data('read-only')
+                        readOnly: this.$('.discussion-module').data('read-only') === true
```

Comment:
You shouldn't need ``=== true``. If it is already a boolean, then the test does nothing. If it isn't already a boolean, then it will always return false.